code/extensions/ContentReviewDefaultSettings.php

```diff
@@ -120,11 +129,19 @@ public function updateCMSFields(FieldList $fields)
         $usersMap = $users->map('ID', 'Title')->toArray();
         asort($usersMap);
 
-        $userField = ListboxField::create('OwnerUsers', _t('ContentReview.PAGEOWNERUSERS', 'Users'), $usersMap)
-            ->setMultiple(true)
-            ->setAttribute('data-placeholder', _t('ContentReview.ADDUSERS', 'Add users'))
-            ->setDescription(_t('ContentReview.OWNERUSERSDESCRIPTION', 'Page owners that are responsible for reviews'));
-
+        if(Member::get()->count() > Config::inst()->get('SiteConfig', 'content_review_gridfield_threshold')) {
+            $gridfieldconfig = GridFieldConfig_RelationEditor::create();
+            $gridfieldconfig->removeComponentsByType(new GridFieldAddNewButton());
+            $userField = GridField::create('OwnerUsers', _t("ContentReview.PAGEOWNERUSERS", "Users"),
+                    $this->OwnerUsers()->Sort('FirstName'), $gridfieldconfig)
```

Comment:
Could you remove the sorting on the OwnerUsers list here? It's not used in the SiteTreeContentReview equivalent and we should really leave it up to the default sort of the Member model